Task6/src/main/java/arrays/ArrayListAndArray.java

```diff
@@ -0,0 +1,60 @@
+package arrays;
+
+import java.util.ArrayList;
+import java.util.Date;
+
+public class ArrayListAndArray {
+    private String[] array =new String[10];
+    private ArrayList<String> list = new ArrayList<String>();
+    long timeArrayAdd100k;
+    long timeListAdd100k;
+
+    public void addStrings(String[] array, String newString){
+        int arrayLength = array.length;
+        int newSize = arrayLength * 2;
+
+        if (arrayLength == newSize){
```

Comment:
Look at this if statement. What it suppose to do? Should else be there?
Hint: there are a bug. Think what method suppose to do. Check array size after **add100kStringsToArray** method.

Reply:
I expect if it will work correctly you will see big difference for array and list 100K insert.

Reply:
done